semcore/add-filter/src/components/AddFilterDropdown.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import createComponent, { Component, Root } from '@semcore/core';
+import Dropdown from '@semcore/dropdown';
+import { AddFilterItemProps } from '../AddFilter.types';
+import { FilterTrigger } from '@semcore/base-trigger';
+
+type AsPropsTypeWithHandlers<T> = T & {
+  onChange: (v: any) => void;
+  onClear: () => void;
+};
+
+class AddFilterDropdownRoot extends Component<AddFilterItemProps> {
+  static displayName = 'AddFilterDropdown';
```

Comment:
maybe just `Dropdown`? Because the name will combine with parents name

Reply:
left as is, as dicsussed